lib/constants.js

```diff
@@ -41,6 +41,14 @@ module.exports.TRANSACTION_STATUS = {
   PENDING_VALIDATION: '16'
 };
 
+module.exports.TRANSACTION_INT_STATUS = {
+  SUCCESS: '0',
+  IN_PROGRESS: '4',
+  ERROR: '6',
+  CANCELLATION_SUCCESS: '7',
+  PENDING_VALIDATION: '16'
```

Comment:
I would put 
```suggestion
  PENDING_VALIDATION_OR_FINALIZATION: '16'
```
(same as the doc)